Classes/gitx_askpasswd_main.m

```diff
@@ -45,7 +45,7 @@ -(NSPanel*)	passwordPanel
 													  backing: NSBackingStoreBuffered defer: NO];
 		[mPasswordPanel setHidesOnDeactivate: NO];
 		[mPasswordPanel setLevel: NSFloatingWindowLevel];
-		[mPasswordPanel setTitle: @"GitX SSH Remote Login"];
+		[mPasswordPanel setTitle: NSLocalizedString(@"GitX SSH Remote Login", @"Title for password panel in command line tool")];
```

Comment:
I had a hunch someone would have done this 😆 .

Runtime-replacing `-[NSBundle localizedStringForKey:value:table:]` would make that work 😁 .

BTW, hats up @ssp for adding those `NSLocalizedString`s !